doc/format-run.md

```diff
@@ -175,6 +175,22 @@ ordinary index type.
 The type of index in use is metadata that must be known for the whole LSM table
 handle. It does not vary per run.
 
+Independently of the concrete index serialization format, the type and the
+version of the format are stored at the beginning as a 32-bit number called the
+type–version indicator. A type–version indicator has the following shape, where
+bit numbers refer to significance:
+
+* Bits 0–7 contain the version number.
+* Bits 8–15 contain a number denoting the type of the index.
+* Bits 16–31 are zero.
+
+As a result, a type–version indicator can be written in hexadecimal with only
+4-digits of which the first two denote the type and the last two constitute the
```

Comment:
Replace first and last with most and least significant

Reply:
It is about how this number is written, and in writing there are first and last digits. Well, if you place the same written representation of a number into some Arabic text, for example, first becomes last and *vice versa*. Maybe I should indeed change this.

Reply:
Done in 17a3c914063c83f0680ebef15acce01e510f7470.